.changeset/great-spies-exist.md

```diff
@@ -0,0 +1,51 @@
+---
+'@xstate/graph': major
+---
+
+Coverage can now be obtained from a tested test model via `testModel.getCoverage(...)`:
+
+```js
+import { configure, createTestModel } from '@xstate/test';
+import {
+  coversAllStates,
+  coversAllTransitions
+} from '@xstate/test/lib/coverage';
```

Comment:
is there any advantage for this here? I mean - cant we just reexport everything from the main entrypoint?

Reply:
Will add a PR changing this

Reply:
Resolved to remove coverage functions